src/main/kotlin/rmc/HTTP requests vehicle.http

```diff
@@ -21,7 +21,7 @@ Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJodHRwOi8vM
 ###
 // CREATE A VEHICLE without auth
 
-POST http://0.0.0.0:8080/createVehicle
+POST http://0.0.0.0:8080/vehicle/createVehicle
```

Comment:
Deze endpoint zonder auth is ook om te testen?

Reply:
jazeker